src/components/reusable/table/table-expanded-row.scss

```diff
@@ -0,0 +1,28 @@
+:host {
+  display: table-row;
+  block-size: 0;
+  transition: height 150ms ease-out;
+
```

Comment:
Generally we are using 400ms for smooth animation. You can also check if its add more smoothing in animation.

Reply:
The transition doesn't actually do anything, since no explicit height is being set. We don't want to animate height either as it's a property that causes a layout change. Can just remove it.

Reply:
I can dynamically set the height using JS.  So if we're not animating height. Do you have any suggestions?

Reply:
You won't necessarily know the content height. The content could be loaded asynchronously. I would say just have no open/close animation, same as Accordion.

Reply:
Ok. Discussed with Matthew that we'll have no animation for row exapdning.